Sources/Routing/RouteBuilder.swift

```diff
@@ -7,6 +7,7 @@ public typealias WebSocketRouteHandler = (Request, WebSocket) throws -> Void
 /// Used to define behavior of objects capable of building routes
 public protocol RouteBuilder: class {
     func register(host: String?, method: Method, path: [String], responder: Responder)
+    func register(host: String?, method: Method, path: [String], metadata: [String: Any], responder: Responder)
```

Comment:
Since you're introducing a secondary implementation, it's sensible to provide a default implementation for this so other Routers' their API aren't breaking. Even if it means that you don't use the `metadata`.

Reply:
@Joannis this counts as a default implementation, right? https://github.com/vapor/routing/pull/34/files#diff-80f2ca0206c7bebeeaea410d4e76af49R47

Reply:
Ignore that comment, then :) @tanner0101 ?